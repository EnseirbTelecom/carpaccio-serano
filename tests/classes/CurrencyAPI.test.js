/* eslint-env jest */

const CurrencyAPI = require('../../src/ServiceClasses/CurrencyService/CurrencyAPI')

test('fetch the good reponse', async () => {
  const reponse = await CurrencyAPI.getLatestExchangeRate()
  const json = await reponse.json()
  console.log(json)
  expect(json).toMatchSnapshot()
})
