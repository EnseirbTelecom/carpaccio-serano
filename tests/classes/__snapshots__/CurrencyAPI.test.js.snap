// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetch the good reponse 1`] = `
Object {
  "base": "EUR",
  "date": "2021-01-20",
  "rates": Object {
    "AUD": 1.5666,
    "BGN": 1.9558,
    "BRL": 6.439,
    "CAD": 1.5379,
    "CHF": 1.0778,
    "CNY": 7.8292,
    "CZK": 26.125,
    "DKK": 7.4382,
    "GBP": 0.88563,
    "HKD": 9.3804,
    "HRK": 7.5635,
    "HUF": 357.38,
    "IDR": 17004.08,
    "ILS": 3.9373,
    "INR": 88.368,
    "ISK": 157.2,
    "JPY": 125.62,
    "KRW": 1333.62,
    "MXN": 23.7719,
    "MYR": 4.8949,
    "NOK": 10.323,
    "NZD": 1.6965,
    "PHP": 58.122,
    "PLN": 4.5322,
    "RON": 4.8738,
    "RUB": 89.085,
    "SEK": 10.1383,
    "SGD": 1.6054,
    "THB": 36.315,
    "TRY": 9.0111,
    "USD": 1.2101,
    "ZAR": 18.1101,
  },
}
`;
